rest-backend/pom.xml

```diff
@@ -346,12 +346,12 @@
 							</execution>
 						</executions>
 					</plugin>
-					<!--  Run the following to fix all JavaDoc: "mvn -DdefaultAuthor=SAP -X -DfixTags=param,return,throws -P gradle clean compile org.apache.maven.plugins:maven-javadoc-plugin:3.1.1:fix"
+					<!--  Run the following to fix all JavaDoc: "mvn -DdefaultAuthor=SAP -X -DfixTags=param,return,throws -DignoreClirr=true -P gradle clean compile org.apache.maven.plugins:maven-javadoc-plugin:3.3.0:fix"
```

Comment:
What does ignoreClirr do?

Reply:
It ignores the [Clirr tool](http://clirr.sourceforge.net/), which _checks Java libraries for binary and source compatibility with older releases_. With Clirr enabled, the properties `maven.compiler.source` and `target`, defined in the root's `pom.xml` are ignored, which results in the following compile error when running the fix goal of the `maven-javadoc-plugin`:

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile (default-compile) on project shared: Compilation failure: Compilation failure: 
[ERROR] Source option 5 is no longer supported. Use 6 or later.
[ERROR] Target option 1.5 is no longer supported. Use 1.6 or later.
```